main.js

```diff
@@ -56,28 +70,18 @@ renderToDom('#introContainer',domString)
 const studentInfoCard = (array) => {
   let domString = " "
   for(let student of array){
-    if(student.expelled === false)
+    if(student.expelled === false || true)
```

Comment:
```suggestion
    if(!student.expelled)
```
We can use the bang here. We also want to remove the `|| true` because then this condition will always result in true.